lib/rules/arrow-parens.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @fileoverview Rule to require parens in arrow function arguments.
+ * @author Jxck
+ * @copyright 2015 Jxck. All rights reserved.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+    var message = "arrow function argument should wrap with parens";
```

Comment:
We try to be formal with our message, so to clean up:

"Expected parentheses around arrow function argument."